x-pack/plugins/ingest_manager/server/routes/datasource/index.ts

```diff
@@ -58,4 +60,14 @@ export const registerRoutes = (router: IRouter) => {
     },
     updateDatasourceHandler
   );
+
+  // Delete
+  router.post(
+    {
+      path: DATASOURCE_API_ROUTES.DELETE_PATTERN,
+      validate: DeleteDatasourcesRequestSchema,
+      options: { tags: [`access:${PLUGIN_ID}`] },
```

Comment:
It's not implemented anywhere for now, but we should make a difference between read and write API routes here it should be only accessible for user with write access: `access:ingestManager_write`

Reply:
I think this would be good to fix across all routes, deferring for now